src/components/SearchRecords.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Message from './message';
+import { Typography } from '@material-ui/core';
+import { makeStyles } from '@material-ui/styles';
+
+const useStyles = makeStyles({
```

Comment:
Very good. I see you've experimented with `makeStyles`. Do you like it? I'm quite fond of it.

Reply:
This is really growing on me, lol.